assingment6.txt

```diff
@@ -0,0 +1,10 @@
+Can Parlar
+
+1. The declaration of arr[num] needs to be after scanning num from the user.
```

Comment:
Right! otherwise it causes a segmentation fault! you did forget to note there's no `return 0;`